cold-wallet/wallet/cold-wallet.ts

```diff
@@ -97,6 +98,14 @@ export class ColdWallet {
     }
   }
 
+  public async signMessage(message: string): Promise<string> {
+    if (!this.wallet) throw new Error('Wallet is not initialized');
+
+    const account = await this.wallet.get(0);
+    const signedMessage = await signAsync(message, await account.privateKey(), true, this.network.messagePrefix);
```

Comment:
How can we guarantee that this functionality is not misused to sign transactions? Would this be possible?

Reply:
Just checked, you can't sign a tx with it